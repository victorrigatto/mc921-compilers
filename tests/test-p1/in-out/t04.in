int main(){
    int i = 0; \\ init the variable
    return 0;
}
