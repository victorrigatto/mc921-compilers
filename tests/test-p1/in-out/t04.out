LexToken(INT,'int',1,0)
LexToken(ID,'main',1,4)
LexToken(LPAREN,'(',1,8)
LexToken(RPAREN,')',1,9)
LexToken(LBRACE,'{',1,10)
LexToken(INT,'int',2,16)
LexToken(ID,'i',2,20)
LexToken(EQUALS,'=',2,22)
LexToken(INT_CONST,'0',2,24)
LexToken(SEMI,';',2,25)
Lexical error: Illegal character '\\' at 2:16
Lexical error: Illegal character '\\' at 2:17
LexToken(ID,'init',2,30)
LexToken(ID,'the',2,35)
LexToken(ID,'variable',2,39)
LexToken(RETURN,'return',3,52)
LexToken(INT_CONST,'0',3,59)
LexToken(SEMI,';',3,60)
LexToken(RBRACE,'}',4,62)
